### PR TITLE
Safer usage of env variables.

### PR DESCRIPTION
**Pull request by deconstructionalism**
- using the `dotenv` package alone was appending all environmental variables
  from the build machine to the build, including values from other
  potentially sensitive processes on the host machine. This is not good!
- using `dotenv-webpack`, only env values that are actually referenced in the
  app are bundled in the build.
- yes indeed, the env values that are used in the app are bundled into
  the build. There is no easy way to avoid this with a client-side
  rendered app. However, the values at least are not easily available on GitHub,
  a place where it looks unprofessional to have those values saved in a file,
  and a place  which is full of people who actually know how to take advantage
  of those secret values!

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -61,12 +53,12 @@ module.exports = {
...
   plugins: [
-    // provide values to application from process.env
-    new webpack.DefinePlugin({
-      'process.env': Object.assign({ 
-        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
-      }, env)
-    }),
+    // provide values to application from process.env, only providing
+    // values referenced in codebase, which is safer than providing all env
+    // values
```
Comment (kangabell) [inline webpack.config.js]: is this line a typo?

Comment (deconstructionalism) [reply webpack.config.js]: nope! that's how I meant it

Comment (kangabell) [reply webpack.config.js]: oh okay it just looked weird because this comment block is wrapping differently than the one up by line 19. 🤷‍♀️

**Comment by kangabell** [comment]
approved and deployed!

**Pull request merged**